tests/integration/targets/terraform/vars/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+# Terraform version that will be downloaded
+terraform_version: 1.1.7
+
+# Architecture of the downloaded Terraform release (needs to match testing platform)
+terraform_arch: linux_amd64
```

Comment:
How about composing this from Ansible facts?

Reply:
It can be done, but we will need to maintain a mapping of the arch names that Ansible uses and whatever Hashicorp uses for their releases, as they're most likely not the same.

Reply:
Ansible usually provides what you can get from `uname`, and almost everyone on this planet uses the same identifiers :) Also it suffices if it works for the architectures in CI (which is basically what you have here as a constant), if someone finds out it doesn't work for another one they can still provide a fix.

Reply:
I have added a simple arch lookup. I cross-checked it for MacOS and Linux x86_64 platforms. Others can add additional ones if needed.